Dockerfile

```diff
@@ -12,34 +12,41 @@ WORKDIR /nativefier
 # Add sources
 COPY . .
 
+# Give everything to node
```

Comment:
In comments, please document the "why", not just the "what". The "what" alone doesn't need much commenting when the code is clear/easy like it is here.

Here's a short talk I like a lot on the subject. It's about Git commit messages, but applies well to comments too. [PyCon 2014 / Greg Ward - Documenting history, or how to write great commit messages: say *why*, not what](https://www.youtube.com/watch?v=Jb6ij4eRu6c#t=378)